src/styles/_components/_inputs.scss

```diff
@@ -32,9 +32,14 @@ $checkbox-size: 24px;
             background-color: color('input', 'focus');
         }
     }
+
+    &--full-width {
+        width: 100%;
+    }
 }
 
 @mixin input() {
+    @include height-modifiers();
     height: $input-height;
```

Comment:
aren't these two lines in reverse? since we want to give preference to the modifier if present.